lib/identity_cache/query_api.rb

```diff
@@ -108,7 +108,7 @@ def prefetch_associations(associations, records)
 
       def raise_if_scoped
         if current_scope
-          IdentityCache.logger.error("#{name} has scope: #{current_scope.where_values_hash.inspect}")
+          IdentityCache.logger.error("#{name} has scope: #{current_scope.to_sql} (#{current_scope.values.keys})")
```

Comment:
`current_scope.values.keys` does not seems to give out a lot of interesting information?